src/content/developers/docs/scaling/optimistic-rollups/index.md

```diff
@@ -1,36 +1,213 @@
 ---
 title: Optimistic Rollups
-description: Introduction to optimistic rollups
+description: An introduction to optimistic rollups, a scaling solution used by the Ethereum community. 
 lang: en
 sidebar: true
 ---
 
+Optimistic rollups sit in parallel to the main Ethereum chain on layer 2. They can offer improvements in scalability because they reduce computation on Ethereum’s base layer. After processing transactions off-chain, optimistic rollups publish the results on Ethereum, but don't re-execute the transactions on Mainnet. 
```

Comment:
Just a general thought/note - I like framing L2s as "extending" the base layer, e.g. https://twitter.com/toghrulmaharram/status/1534359799311253505?s=20&t=VxDjdiP67hBtXKgvB-oOkQ

Reply:
> Just a general thought/note - I like framing L2s as "extending" the base layer, e.g. https://twitter.com/toghrulmaharram/status/1534359799311253505?s=20&t=VxDjdiP67hBtXKgvB-oOkQ

That makes sense. I'll edit it to reflect the suggestion. The text was in the original page, so I wanted to leave it (and just flesh it out a bit), but I guess better clarifying rollups wouldn't hurt.